emperor/support_files/js/color-view-controller.js

```diff
@@ -182,6 +186,16 @@ ColorViewController.getDiscreteColor = function(index, map){
   return ColorViewController._discreteColormaps[map][index];
 };
 
+/* Change the color for a set of plottables - group: list of plottables */
+ColorViewController.setPlottableAttributes = function(scope, color, group){
```

Comment:
Not sure how to create the scope and the group. Note that it didn't have one before so not sure what to check. Could you help?